### PR TITLE
Simplify the subprocess and rearrange the file structure

### PR DESCRIPTION
**Pull request by BECATRUE**
As I wrote in #11, I implemented two gaols; simplifying the subprocess and rearranging the file structure.

In `run_command()` of `cmdtools.py`, the meaning of each argument is
- `capture_output`: If true, `stdout` and `stderr` will be captured.
- `shell`: If true, commands will be executed through the shell.
- `check`: If true, it checks a non-zero exit code. Also, if you don't set it, it causes a Pylint warning.
- `text`: If true, the result will be stored as a string. Otherwise, it will be stored as a bianry.

An example is as follows.

### CODE REVIEW

File: iquip/cmdtools.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+"""Utility module for handling the command line."""
+
+import subprocess
+
+def run_command(cmd):
```
Comment (kangz12345) [inline iquip/cmdtools.py]: Please annotate the parameter and return types.

**Review by kangz12345 — CHANGES_REQUESTED** [review]

File: iquip/cmdtools.py (hunk 1 above)
Comment (BECATRUE) [reply iquip/cmdtools.py]: Oh, I forgot using type hinting! I will update it.

**Comment by BECATRUE** [comment]
I applied it. @kangz12345

**Review by kangz12345 — APPROVED** [review]
LGTM

**Pull request merged**